### ISSUE TITLE
[telegram] sendTelegram to n chatIds

### ISSUE THREAD

**Issue by Rodelkoenig** [openhab/openhab-addons#8018]
**Goal:**
As a user I want a new possibility to send a telegram-message to several chatIds with a single line.

**Comment by Rodelkoenig** [comment]
I would try to implement this as my first contribution here, if it makes sense to you.
Main goal was to reduce lengthy rules if a message is only for e.g.  5 out of 6 chatids/persons which are configured to my bot.

I thought of a method sendTelegram(String msg, Long... chatIds).

**Comment by J-N-K** [comment]
@ZzetT I would suggest to change the first parameter of the thing actions to a `String` (the `Long` version can stay as deprecated for backward compatibility). Then we could just line-up chat-Ids like `"123345679,567891234" and split that in the actions. WDYT?

**Comment by ZzetT** [comment]
Would this mean that we have to "mirror"(copy) all existing actions that have currently the chatId as the first parameter? If I search for "@Nullable Long chatId", I count 16 methods. I think 7 of them are public actions.

Currently, there are actions with the chatId (long) as the first parameter and without. The methods without the chatId send the message to all chatIds configured for a bot. We might now run into trouble if we use a String as the first parameter for a chatId.
For example the existing signature `sendTelegram(ThingActions actions, @Nullable String format, @Nullable Object... args)`
couldn't be distinguished anymore from a new method signature `sendTelegram(ThingActions actions, @Nullable String chatId, @Nullable String format, @Nullable Object... args)`. You would probably get a compile error. Only way out is to rename the method instead of overloading it, right?

**Comment by J-N-K** [comment]
Correct. Maybe `sendTelegramTo` (so post-fixed with `To`) would be a good naming. We could then leave the old  methods with `Long`  like they are.

**Comment by ZzetT** [comment]
Yes, I'm fine with that.

I just fear a bit that the number of APIs could explode in the future :) Maybe a "command like" API could help to support more features. What I mean is an API like `sendTelegram(String command)` where you can pass a command like "sendMessage -chatIds <chatIds> -message <message> -featureX something". Then we would need to implement only one API and could be very flexible with the command structure and parse it with an argument parser that translates all available options into objects for the Telegram library. WDYT?

But of course, in the context of this issue, I'm fine with adding a new API.
